src/main/resources/application.yml

```diff
@@ -46,6 +46,7 @@ govNotify:
 invite:
   url: ${INVITE_SIGNUP_URL:http://localhost:8080/signup/%s}
   validityInSeconds: ${INVITE_VALIDITY:259200}
+  durationAfterReRegAllowedInSeconds: ${REREG_VALIDITY:86400}
```

Comment:
Unit is included in the config variable name in the app which is useful but could be useful to also include it in the environment variable name for future people changing the property's value - `REREG_VALIDITY_SECONDS` would make it very obvious.

Reply:
DONE.
Review comment is applied. Please refer PR:
https://github.com/Civil-Service-Human-Resources/identity-service/pull/196/files